packages/cli/src/config/schema.ts

```diff
@@ -439,7 +439,7 @@ export const schema = {
 				env: 'QUEUE_RECOVERY_INTERVAL',
 			},
 			gracefulShutdownTimeout: {
-				doc: 'How long should n8n wait for running executions before exiting worker process (seconds)',
+				doc: '[DEPRECATED] (Use N8N_GRACEFUL_SHUTDOWN_TIMEOUT instead) How long should n8n wait for running executions before exiting worker process (seconds)',
```

Comment:
Should we update [the docs](https://docs.n8n.io/hosting/environment-variables/environment-variables/) as well?

Reply:
Yes: https://github.com/n8n-io/n8n-docs/pull/1792